package.json

```diff
@@ -9,7 +9,7 @@
 		"index.d.ts"
 	],
 	"engines": {
-		"node": ">=10"
+		"node": ">=12.3.0"
```

Comment:
this is a breaking change - is it needed?

Reply:
Yep, it is required for using Readable class.

Reply:
I'm checking if there is an alternative.

Reply:
This is the eslint error when using node=10: 
```
The 'stream.Readable.from' is not supported until Node.js 12.3.0 (backported: ^10.17.0). The configured version range is '>=10'.
```

Reply:
An alternative might be using `readable-stream`.

However, dropping node 10 which has been EOL since April 2021 is probably fine. 12 is also EOL now, so probably fine to drop that as well